opentelemetry-otlp/examples/basic-otlp/Cargo.toml

```diff
@@ -12,5 +12,5 @@ opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio
 opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
 opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
 tokio = { version = "1.0", features = ["full"] }
-opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log"}
+opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log", default-features = false}
```

Comment:
Thanks for the fix. Can you also add similar change for [logs_basic](https://github.com/open-telemetry/opentelemetry-rust/blob/main/examples/logs-basic/Cargo.toml)